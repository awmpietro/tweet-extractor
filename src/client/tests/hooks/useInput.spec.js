import React from 'react';
import useInput from '../../hooks/useInput';
import renderer from 'react-test-renderer';

describe('useInput', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<useInput initialValue={""} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});