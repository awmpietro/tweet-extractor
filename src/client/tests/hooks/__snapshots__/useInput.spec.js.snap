// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`useInput snapshot renders 1`] = `
<useInput
  initialValue=""
/>
`;
